libs/ui/src/lib/activities-table/activities-table.component.ts

```diff
@@ -37,6 +38,7 @@ export class ActivitiesTableComponent implements OnChanges, OnDestroy {
   @Input() hasPermissionToImportActivities: boolean;
   @Input() hasPermissionToOpenDetails = true;
   @Input() locale: string;
+  @Input() pageSize = Number.MAX_SAFE_INTEGER;
```

Comment:
I would use a default page size of 50 for every table:

```
@Input() pageSize = DEFAULT_PAGE_SIZE;
```

Add `DEFAULT_PAGE_SIZE=50` to `libs/common/src/lib/config.ts`

Reply:
Fixed